build/android/patches/irrlicht-back_button.patch

```diff
@@ -1,14 +1,6 @@
---- irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp	2014-06-03 20:56:21.289559503 +0200
-+++ irrlicht/source/Irrlicht/Android/CIrrDeviceAndroid.cpp.orig	2014-06-03 20:57:39.281556749 +0200
-@@ -423,6 +423,7 @@
- 			}
- 			
- 			device->postEventFromUser(event);
-+			status = 1;

```

Comment:
I'm not a programmer. You ought to know what's best. But this raises some problems, the players uncomfortable, and do not welcome Google